### ISSUE TITLE
Adjust timeout for "should not be listed" steps

### ISSUE THREAD

**Issue by phil-davis** [owncloud/phoenix#4231]
There is a step:
`Then user "User Name" should not be listed in the collaborators list on the webUI`

When that step runs, it is looking for the non-existence of something on the UI. So it does need to wait "a while" to feel confident that the element is not there. But currently it waits for 20 seconds. That makes scenarios very slow, specially ones that do 2 of these tests. For example: 

https://drone.owncloud.com/owncloud/ocis/860/31/7
```
    And user "user3" should have received a share with these details:
      │ field       │ value                        │
      │ uid_owner   │ user1                        │
      │ share_with  │ user3                        │
      │ file_target │ /simple-folder               │
      │ item_type   │ folder                       │
      │ permissions │ read, delete, update, create │
    But user "Regular User" should not be listed in the collaborators list on the webUI
Timed out while waiting for element <//*[contains(@class, "files-collaborators-collaborator-name") and .="Regular User"]/ancestor::*[contains(concat(" ", @class, " "), " files-collaborators-collaborator ")]> to be present for 20000 milliseconds. - expected "found" but got: "not found" (20023ms)
undefined    And as "user0" folder "simple-folder (2)" should not exist
    And user "User Four" should not be listed in the collaborators list on the webUI
Timed out while waiting for element <//*[contains(@class, "files-collaborators-collaborator-name") and .="User Four"]/ancestor::*[contains(concat(" ", @class, " "), " files-collaborators-collaborator ")]> to be present for 20000 milliseconds. - expected "found" but got: "not found" (20009ms)
undefined    And as "user4" folder "simple-folder (2)" should not exist
```
from feature files like https://github.com/owncloud/phoenix/blob/master/tests/acceptance/features/webUISharingFolderPermissionMultipleUsers/shareFolderWithMultipleUsers.feature

Reduce this - will 5 seconds be enough? Or 10 seconds?

**Comment by phil-davis** [comment]
See #4232

**Comment by individual-it** [comment]
fixed by #4238
